src/commands/edit.ts

```diff
@@ -282,22 +282,50 @@ export async function replaceCharacters(
  *
  * @keys `&` (kakoune: normal)
  */
-export function align(
-  _: Context,
-  selections: readonly vscode.Selection[],
+export function align(_: Context, fill: Argument<string> = " ") {
+  return edit((builder, selections) => {
+    const sortedSelections = Selections.sort(Direction.Forward, [...selections]);
+
+    // Group selections by 'column', nth column being nth selections of each line.
+    const selectionByColumn: vscode.Selection[][] = [];
+    let currentLine: number | undefined = undefined;
```

Comment:
To avoid having `currentLine` change types, maybe we could initialize it as `0`, since it results as a no-op in the condition that checks `selection.start.line !== currentLine` below.

Reply:
Sure, sounds good to me as well. `0` wasn't meant to mean "cannot be a line", but AFAIK things work normally if you start at `0` as well. Doesn't matter that much.